specification/postgresql/Microsoft.OrionDB.Management/examples/2025-08-01-preview/Clusters_CreateOrUpdate.json

```diff
@@ -0,0 +1,203 @@
+{
+  "operationId": "Clusters_CreateOrUpdate",
+  "title": "Create or update an OrionDB cluster",
+  "parameters": {
+    "subscriptionId": "ffffffff-ffff-ffff-ffff-ffffffffffff",
+    "resourceGroupName": "TestGroup",
+    "orionDbClusterName": "testcluster1",
```

Comment:
"orionDbClusterName" => "orionDBClusterName" to be consistent with the naming everywhere else. Obviously, this needs to be applied also on the model behind the ARM controller, so that this matches with what the REST API will really return.